goblet/app.py

```diff
@@ -49,6 +63,30 @@ def local_func(request):
 
             setattr(sys.modules[module_name], local, local_func)
 
+    def deploy(self, skip_function=False, only_function=False, config={}, force=False):
+        source = None
+        if not skip_function:
+            log.info(f"preparing to deploy with backend {self.backend_class.__name__}")
+            source = self.backend_class(self).deploy(force=force, config=config)
+        if not only_function:
+            self.deploy_handlers(source, config=config)
+
+    def destroy(self, all=None):
+        """Destroys http cloudfunction and then calls goblet.destroy() to remove handler's infrastructure"""
+        for k, v in self.handlers.items():
+            log.info(f"destroying {k}")
+            v.destroy()
+        versioned_clients = VersionedClients(self.client_versions)
+        if self.backend.startswith("cloudfunction"):
```

Comment:
should we add the destroy to the backend classes as well. same a deploy?

Reply:
Yep, fixed!